recipe/meta.yaml

```diff
@@ -14,15 +14,20 @@ source:
     - fix-install.patch  # [win]
 
 build:
-  number: 1
-  skip: true  # [vc<14]
+  number: 2
+  run_exports:
+    # Symbols were last removed on update from 1.7 to 1.8
+    # https://abi-laboratory.pro/index.php?view=timeline&l=libpcap
+    - {{ pin_subpackage(name, max_pin='x.x') }}
```

Comment:
I am following the pinning from [conda-forge-pinning](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/1e5c1180072839298aad08ba4843ccab27b15f33/recipe/conda_build_config.yaml#L228-L229)